includes/class-wc-stripe-smart-payment-method-defaults.php

```
@@ -0,0 +1,595 @@
+	 *
+	 * @param string $plugin Activated plugin path.
+	 * @return void
+	 */
+	public function maybe_apply_subscriptions_activation_backfill( $plugin ): void {
+		$is_subscriptions_plugin = is_string( $plugin ) && false !== strpos( $plugin, 'woocommerce-subscriptions' );
+		if ( ! $is_subscriptions_plugin && ! $this->is_subscriptions_enabled() ) {
+			return;
+		}
+
+		$this->apply_subscriptions_activation_backfill();
+	}
```

PHPMD (2.15.0)

[error] The method maybe_apply_subscriptions_activation_backfill is not named in camelCase. (undefined)

(CamelCaseMethodName)